tests/ownable.tests.ts

```diff
@@ -0,0 +1,166 @@
+import { consts } from './constants'
+import {bnArg, expect, fromSigner, setupContract} from './helpers'
+
+describe('MY_OWNABLE', () => {
+  async function setup() {
+    return setupContract('my_ownable', 'new')
+  }
+
+  it('PSP 1155 - mint works', async () => {
```

Comment:
You also need to verify `is_approved_for_all`, `set_approval_for_all`, and both "safe" functions(for that you need to deploy a mock Receiver and verify that on_received was called).

About safe methods. Could you also add the same test for PSP721 please?